configuration/multiple_kernels.rst

```diff
@@ -148,10 +148,10 @@ The ``bin/console`` script used to run Symfony commands always uses the default
 to execute console commands using the new kernel, duplicate the ``bin/console``
 script and rename it (e.g. ``bin/api``).
 
-Then, replace the ``AppKernel`` instantiation by your own kernel instantiation
-(e.g. ``ApiKernel``) and now you can execute commands using the new kernel
-(e.g. ``php bin/api cache:clear``) Now you can use execute commands using the
-new kernel.
+Then, replace the ``AppKernel`` instant by your own kernel instant (e.g.
+``ApiKernel``) and now you can execute commands using the new kernel (e.g.
+``php bin/api cache:clear``) Now you can use execute commands using the new
```

Comment:
and now you can...

and again 

Now you can....

BTW there is a dot missing before „Now“ as it starts a new sentence

Reply:
Or the last sentence should be removed, looks like a duplicate